docs/api/_index.md

```diff
@@ -431,6 +433,14 @@ GET /ruler_ring
 
 Displays a web page with the ruler hash ring status, including the state, healthy and last heartbeat time of each ruler.
 
+### Ruler rules
+
+```
+GET /ruler/rules
```

Comment:
We're listing rule groups, should we call the endpoint that way? Ie. `/ruler/rule_groups`.

More importantnly, let's mention that this endpoint is not part of ruler-API which is enabled by the flag, but is always available. It should not be exposed to end users.